src/utils.jl

```diff
@@ -20,3 +20,18 @@ function wcs_to_celestial_frame(wcs::WCSTransform)
 
     return radesys
 end
+
+"""
+    pad_edges(array_in::Matrix{T}) where {T}
+
+Pads a given array and creates a border with edge elements.
+"""
+function pad_edges(array_in::Matrix{T}) where {T}
+    image = Matrix{T}(undef, size(array_in)[1] + 2, size(array_in)[2] + 2)
```

Comment:
Are you sure that it's safe to initialise to a random array? Are we sure that all elements are safely overwritten? Or was it better to set to zero, like you did before? Just asking